Sources/XCParseCore/XCResultToolCommand.swift

```diff
@@ -100,6 +103,14 @@ open class XCResultToolCommand {
             let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
+
+        public override func run() -> ProcessResult? {
+            let superResult = super.run()
+            if superResult == nil { return nil }
+            guard let timestamp else { return superResult }
+            try? FileManager.default.setAttributes([.modificationDate: timestamp as Any], ofItemAtPath: outputPath)
```

Comment:
Should we also consider modifying the creationDate as well to match the timestamp?

Reply:
Good thinking! However, it seems like the creation date automatically/implicitly gets set to the modification date even without us setting it:

![Screenshot 2023-04-19 at 12 50 21](https://user-images.githubusercontent.com/144932/233053840-22b0d5ac-e803-43b1-b688-99bd344cc376.png)


I'd be happy to set it explicitly if you'd prefer though!

Reply:
Was more worried on the case of if FileManager wasn't handling this and the date created was a later date than date modified; no need to make it explicit.